cmd/make.go

```diff
@@ -208,6 +208,6 @@ func init() {
 		},
 	}
 
-	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota-gb", 0, "Specifies the maximum size of the share in gigabytes (GiB), 0 means you accept the file service's default quota.")
+	makeCmd.PersistentFlags().Uint32Var(&rawArgs.quota, "quota-gb", 0, "Specifies the maximum size of the share in gigabytes (GiB), 0 means you accept the file service's default quota. If `ShareSizeLimitReached` error is hit, increase quota and call resume command.")
```

Comment:
This doc will only be seen by customers if they happen to be creating the share with AzCopy and they happen to be reading the help docs. 

Would it make more sense to catch this error in ste and print a helpful log message?

Reply:
It seems like this specific error is not included in the AzCopy or Azure SDK for Go. I opened an [ issue ](https://github.com/Azure/azure-sdk-for-go/issues/23833) to address this, it has not been merged in yet

Reply:
I dont think you need the specific error in the SDK, temporarily you can just use a string as an input to the HasErrorCode function.